src/odom_predictor.cpp

```diff
@@ -19,8 +19,17 @@ OdomPredictor::OdomPredictor(const ros::NodeHandle& nh,
 
   odom_pub_ = nh_private_.advertise<nav_msgs::Odometry>("predicted_odometry",
                                                         kROSQueueLength);
+  odom2_pub_ = nh_private_.advertise<nav_msgs::Odometry>("predicted_odometry_compare",
+                                                         kROSQueueLength);
+
   transform_pub_ = nh_private_.advertise<geometry_msgs::TransformStamped>(
       "predicted_transform", kROSQueueLength);
+
+  frame_id_ = "odom";
```

Comment:
Can you get these from ROS params please?

Reply:
Done!